cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/async/GetBrokerStatsRunnable.java

```diff
@@ -28,7 +29,8 @@
                          ClusterLoadParameters parameters,
                          KafkaCruiseControlConfig config) {
     super(kafkaCruiseControl, future);
-    _time = parameters.time();
+    _start = parameters.startMs() != null ? parameters.startMs() : -1;
```

Comment:
Can we avoid hard coding `-1`? -- e.g. we may move it to a variable that indicates its function.